src/Lint.jl

```diff
@@ -263,33 +263,53 @@ end
 
 function lintserver(port)
     server = listen(port)
+    println("Server running on port $port ...")
     while true
-      conn = accept(server)
-      @async begin
-        try
-          while true
-            line = readline(conn)
-
-            println(typeof(line))
-            println(ispath(strip(line)))
-            println(strip(line))
-
-            if ispath(strip(line))
-                m = lintfile(strip(line), returnMsgs = true)
-            else
-                error("The string is not a file.")
+        conn = accept(server)
+        @async try
+            println("Connection accepted.")
+            # Get file, code length and code
+            file = strip(readline(conn))
+            println("file: ", file)
+            code_len = parse(Int, strip(readline(conn)))
+            println("code length: ", code_len)
+            code = ""
+            while length(code) < code_len
+                code = string(code, readline(conn))
             end
```

Comment:
If you change the length to be the number of bytes rather than number of characters you could use `readbytes` and read the entire code content at once. This will additionally avoid unnecessary creation of strings.